hedera-node/hedera-app/data/config/bucket-credentials.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Not a blocker, but do we need this file? Looks like a duplicate (or the other could be the duplicate)